ar4_hardware_interface/include/ar4_hardware_interface/ar4_hardware_interface.hpp

```diff
@@ -0,0 +1,79 @@
+// BSD 3-Clause License
+//
+// Copyright 2025 Ekumen, Inc.
```

Comment:
Is this our own code or it comes from another source? If that's the case how we deal with licensing/credits?

Reply:
Good catch. Thanks!